chapters/data/main.tex

```diff
@@ -368,37 +378,40 @@ \subsection{Measuring variation and importance in our cost components}
 
 \begin{figure}[h]
     \centering
-    \includegraphics[width=\imgwidth]{cost_contribution/main.pdf}
-    \caption{Bar chart showing the average contribution of each cost component
-        to the net cost of a spell.}\label{fig:cost_contribution}
+    \includegraphics[width=\imgwidth]{cost_contribution.pdf}
+    \caption{%
+        Average contribution of each cost component to the net cost of a spell
+    }\label{fig:cost_contribution}
 \end{figure}
 
 By inspecting Figure~\ref{fig:cost_contribution}, it is seen that ward, overhead
 and medical (MED) costs are the largest contributors to the net cost of a spell
-by a large margin. When looking across the remaining bars, the contribution is
-substantially smaller for the department-specific cost components. Not only that
-but it appears that the most varied components (from
+by a significant margin. When looking across the remaining bars, the
+contribution is substantially smaller for the department-specific cost
+components. Not only that but it appears that the most varied components (from
 Figure~\ref{fig:cost_variation}) have near negligible average contributions to
 the net cost of a spell.
 
-\begin{figure}[htbp]
-    \centering
-    \includegraphics[width=\linewidth]{cost_bubble_plot/main.pdf}
-    \caption{A bubble plot showing the average contribution to the net cost of a
-    spell along the vertical, and the coefficient of variation for that
-    component as the size of its marker.}\label{fig:cost_bubble_plot}
-\end{figure}
-
 So the question left to be answered is: can these small but highly varied
 components be considered especially important? And what about the other
 components? The midriffs of each of these figures contain many of the same
-components but the relationships are less clear. In order to attain some
-understanding of how these two quantities relate to one another, a bubble plot
-is used. Such a plot allows for three-dimensional data to be displayed in the
-two-dimensional plane; by running their common variable along the horizontal
-axis, both of the quantities can be visualised together by using the vertical
-axis and size as two separate dimensions, as illustrated in
-Figure~\ref{fig:cost_bubble_plot}.
+components but the relationships are less clear. In order to visualise how these
+two quantities relate to one another, a bubble plot is used. Such a plot allows
+for three-dimensional data to be displayed in the two-dimensional plane; by
```

Comment:
In contrast to my comment about the visualisation of the correlation coefficients, this paragraph doesn't niggle at me at all. You're explaining the specific purpose of the chosen visualisation.